integration/experiment/uncore_frequency_sweep/uncore_frequency_sweep.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+#
+#  Copyright (c) 2015, 2016, 2017, 2018, 2019, 2020, Intel Corporation
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions
+#  are met:
+#
+#      * Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in
+#        the documentation and/or other materials provided with the
+#        distribution.
+#
+#      * Neither the name of Intel Corporation nor the names of its
+#        contributors may be used to endorse or promote products derived
+#        from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+'''
+Helper functions for running power sweep experiments.
+'''
+
+import sys
+import argparse
+
+import geopmpy.io
+
+from experiment import launch_util
+from experiment import common_args
+from experiment import machine
+from experiment.frequency_sweep import frequency_sweep
+
+
+def setup_run_args(parser):
+    common_args.setup_run_args(parser)
+    parser.add_argument('--max-core-frequency', dest='max_core_frequency',
+                        action='store', type=float, default=None,
+                        help='the maximum core frequency setting for the sweep')
+    parser.add_argument('--min-core-frequency', dest='min_core_frequency',
+                        action='store', type=float, default=None,
+                        help='the minimum core frequency setting for the sweep')
+    parser.add_argument('--step-core-frequency', dest='step_core_frequency',
+                        action='store', type=float, default=None,
+                        help='the step size in hertz between core frequency settings for the sweep')
+    parser.add_argument("--run-max-turbo", dest="run_max_turbo",
+                        action='store_true', default=False,
+                        help='add extra runs to the experiment at maximum turbo frequency')
```

Comment:
This is identical to the one in frequency_sweep.py now.  Should this be moved to common_args.py?

Reply:
yes, I think it should